### DISCUSSION TITLE
Any idea to close signature help

### DISCUSSION THREAD

**Discussion by Old-Farmer** [LazyVim/LazyVim#2691] [Q&A]
![image](https://github.com/LazyVim/LazyVim/assets/84726450/90bfe57c-002f-440d-9360-e895e45607e2)

I need the feature of auto open signature help in insert mode, but sometime I hope I can manually close it without insert characters, any suggestions?

**Comment by dpetka2001** [comment]
Not exactly what you're asking, but you could try disabling the LSP signature `auto_open` and then use explicitly `<C-k>` to trigger it
```lua
  {
    "folke/noice.nvim",
    opts = {
      lsp = {
        signature = {
          auto_open = {
            enabled = false,
          },
        },
      },
    },
  },
```
I'm not aware of how to do it the way you explicitly asked for, so this is just an alternative until someone else more knowledgeable provides you with exactly what you want.

**Comment by alewis001** [comment]
I just admit I haven't tried this is a long time but if I remember correctly, <C-e> closes the cmp window. Sorry if my memory is paying tricks on me.
